net/pfSense-pkg-freeradius3/files/usr/local/pkg/freeradius.inc

```diff
@@ -1601,18 +1601,22 @@ EOD;
 	}
 }
 
+/* This change cannot be pulled back to RELENG_2_4_4 since it would break there.
+ * see https://redmine.pfsense.org/issues/9897
+ */
 // Gets started from freeradiuseapconf.xml
-function freeradius_get_ca_certs() {
-	global $config;
-	$ca_arr = array();
-	$ca_arr[] = array('refid' => 'none', 'descr' => 'none (auto)');
-
-	if (is_array($config['ca'])) {
-		foreach ($config['ca'] as $ca) {
-			$ca_arr[] = array('refid' => $ca['refid'], 'descr' => $ca['descr']);
-		}
+function freeradius_get_ca_or_certs($type) {
+	$c_arr = array();
+	if ($type == 'ca') {
```

Comment:
This can be simplified a bit since you are passing the same string as `$type` on to `cert_build_list()`, so the whole if block can be changed to something like:

```
$c_arr = array();
$c_arr[] = array('refid' => 'none', 'descr' => 'none (auto)');
$ecdsagood = cert_build_list($type, 'IPsec');
```

Reply:
done